packages/xdl/src/Project.ts

```diff
@@ -365,7 +365,7 @@ export async function getSlugAsync(
     ...options
   }: { mode?: 'production' | 'development'; [key: string]: any } = {}
 ): Promise<string> {
-  const { exp } = getConfig(projectRoot, { skipSDKVersionRequirement: true, mode: options.mode });
```

Comment:
mode is destructured from options on line 364, it doesn't exist in options

Reply:
Yea, isn't that was this PR is fixing?